### PR TITLE
fix: fix order intake tests

### PR DESCRIPTION
**Pull request by civsiv**

### CODE REVIEW

File: packages/openactive-integration-tests/test/templates/common.js [hunk 1]
```diff
@@ -170,16 +170,19 @@ function additionalDetailsRequiredAndSupplied(req) {
           '@type': 'PropertyValue',
           propertyID: 'https://example.com/experience',
           value: 'I\'ve played twice before, but I\'m a quick learner so I hope to keep up!',
+          name: 'Level of experience',
```
Comment (civsiv) [inline packages/openactive-integration-tests/test/templates/common.js:173]: Name required: https://openactive.io/open-booking-api/EditorsDraft/1.0CR3/#schema-propertyvaluespecification

Comment (nickevansuk) [reply packages/openactive-integration-tests/test/templates/common.js:173]: It shouldn't be required for the `PropertyValue`? Ref: https://github.com/openactive/open-booking-api/issues/188

Comment (nickevansuk) [reply packages/openactive-integration-tests/test/templates/common.js:173]: Is the validator throwing up this error?

Comment (civsiv) [reply packages/openactive-integration-tests/test/templates/common.js:173]: @nickevansuk Yep validator erroring (2.0.184 models anyway)

**Pull request merged**